Amazon.IonObjectMapper.Test/IonObjectSerializerTest.cs

```diff
@@ -15,12 +15,6 @@ public class IonObjectSerializerTest
     {
         IonSerializer defaultSerializer = new IonSerializer();
         private IValueFactory valueFactory = new ValueFactory();
-
```

Comment:
How come this test got removed?

Reply:
Reinstated the test. Removed that before because I was using the `Engine` test object for IonSerializerFactory, so for the cases we don't have IonSerializerFactory Dictionary<string, object> parameter, it will throw exception.

Addressed this by creating a new test object.